save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/domain/ProjectCoordinates.kt

```diff
@@ -15,4 +15,18 @@ data class ProjectCoordinates(
      * @return ProjectCoordinates as formatted string "{organizationName}/{projectName}"
      */
     override fun toString() = "$organizationName/$projectName"
+
+    /**
+     * @return true if both [organizationName] and [projectName] are empty, false otherwise
+     */
+    fun consideredEmpty() = organizationName.isEmpty() && projectName.isEmpty()
```

Comment:
why not just `isEmpty`?

Reply:
Problem with Jackson serialization. `isEmpty` is serialized into Boolean variable `Empty`.
Cannot apply `JsonIgnore` annotation as far as `ProjectCoordinates` are in `commonMain`, while Jackson is `jvm`

Reply:
got it